### PR TITLE
899 - Change relevant eth.link to eth.limo URLs

### LINKED ISSUE #899: Change the URL of Charts
Change the URL of Charts to dxstats.eth.limo instead of .link that is right now because sometimes user gets error 522 when trying to get to dxstats.

### PR DESCRIPTION
**Pull request by jorgedclopes**
# Summary
Fixes #899

User gets error 522 when trying to get to dxstats so we are changing all these URLs

# To Test

1. yarn start
2. open in browser
3. verify links

### CODE REVIEW

**Comment by jorgedclopes** [comment]
IPFS Preview Deploy workflow as an issue. Being addressed here
https://github.com/levelkdev/swapr-dapp/pull/901

File: src/utils/uriToHttp.test.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import uriToHttp from './uriToHttp'
 
 describe('uriToHttp', () => {
-  it('returns .eth.link for ens names', () => {
+  it('returns .eth.limo for ens names', () => {
```
Comment (adamazad) [inline src/utils/uriToHttp.test.ts:4]: How does this pass?

Comment (jorgedclopes) [reply src/utils/uriToHttp.test.ts:4]: The 1st argument of `it` is the test's name.
Should have no impact on functionality. Also, that test in particular is comparing data from a place we shouldn't be able to reach resulting in an empty array.
I just did a replace all from `.eth.link` to `.eth.limo`.
I'd keep the change for consistency sake, but I can revert or even do something else with it...

**Review by adamazad — APPROVED** [review]
